### ISSUE TITLE
line for url parameters?

### ISSUE THREAD

**Issue by titaniumbones** [pashky/restclient.el#199]
I feel llike i'm missing something obvious, but I am not able to add query parameters to a GET request url except by including them in the initial request. 

So, for instance: 

```
GET https://q.utoronto.ca/api/v1/courses/64706/users?search_term=Matt
Authorization: Bearer XXXXXXXXXXXXXXXXXXX
```

works fine, but if I want to add the search term elsewhere, I don't see how to do it. In various issues reported here I see something like this:

```
GET https://q.utoronto.ca/api/v1/courses/64706/users
Authorization: Bearer XXXXXXXXXXXXXXXXXX
search_term=Matt
```
But that's not working for me.  Am I missing something fundamental? Thank you!

**Comment by pashky** [comment]
Yes, that’s how HTTP works. GET usually doesn’t have payload and all parameters can go only into URL.

**Comment by shaleh** [comment]
True. However, you could emulate https://httpie.org/doc#querystring-parameters and allow for the parameters to be added separately. This can lead to better readability if there are a bunch of parameters.

**Comment by titaniumbones** [comment]
Request .el also does something similar.
